internals/cli/cmd_services.go

```diff
@@ -29,11 +29,11 @@ about all services if none are specified.
 `
 
 type cmdServices struct {
-	clientMixin
 	timeMixin
 	Positional struct {
 		Services []string `positional-arg-name:"<service>"`
 	} `positional-args:"yes"`
+	client *client.Client
```

Comment:
Space precedes client elsewhere, but here and further down its removed.

Reply:
Good catch (:

Reply:
Here, we have: 1 embedded struct + 1 embedded inline struct + 1 member.
My rule is: only append a space after 2 or more lines of the same category. In this case we don't have it so I didn't add a space.

Reply:
For the record, Fred is on a good line of thinking in his reviews: he's looking for and suggesting organization and consistency. Might be worth paying a bit more attention to his nitpicks.

> My rule is: only append a space after 2 or more lines of the same category. In this case we don't have it so I didn't add a space.

It's nice that you're also looking for consistency, but I don't even understand how to apply this rule given the above. To me it also looks off considering what you had elsewhere. Probably worth rethinking it a bit so people can have more empathy for it.

Reply:
I've updating the spacing in all cases to stay more consistent and simpler: let's group members such as `client` or `parser` on the top, and whatever has to do with CLI options after a blank line in between.